scripts/Dockerfile.test

```diff
@@ -38,6 +39,9 @@ RUN source /build.env \
     && dnf -y update \
     && dnf -y clean all \
     && gem install mdl \
+    && mkdir -p ${GOROOT} \
+    && curl https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-amd64.tar.gz  \
```

Comment:
This makes me sad, it breaks on non-amd64 architectures... But I also do not know of a better way to do this, current Fedora 35 (`:latest`) has golang-1.16.x :disappointed:

Reply:
yes, I borrowed it from https://github.com/ceph/ceph-csi/blob/devel/scripts/Dockerfile.devel#L15. https://github.com/ceph/ceph-csi/issues/2707 is for tracking